packages/compiler/src/core/install.ts

```diff
@@ -15,6 +15,7 @@ export async function installTypeSpecDependencies(
 ): Promise<void> {
   const cmd = process.platform === "win32" ? "npm.cmd" : "npm";
   const child = spawn(cmd, ["install"], {
+    shell: true,
```

Comment:
```suggestion
  const child = spawn("npm", ["install"], {
    shell: process.platform === "win32",
```

i think we could do that at least